src/fill_pg_plugin.cpp

```diff
@@ -808,11 +763,11 @@ struct fpg_session : connection_callbacks, std::enable_shared_from_this<fpg_sess
             auto                     bin = opq.get();
             converter.to_sql_values(bin, *abi_type.as_struct(), values);
             // Get rid of producer 3, confirmed 4, schedule_version 8, new_producers 9, header extensions 10
-            values.erase(values.begin() + 10);
-            values.erase(values.begin() + 9);
-            values.erase(values.begin() + 8);
-            values.erase(values.begin() + 4);
-            values.erase(values.begin() + 3);
+	    values.erase(values.begin() + 3);
+	    values.erase(values.begin() + 4);
+	    values.erase(values.begin() + 8);
+	    values.erase(values.begin() + 9);
+	    values.erase(values.begin() + 10);
```

Comment:
@AleksandrAksenov I didn't finish reviewing your changes yet, but are you sure this works? If you remove the 4th element first, the 10th element will be different by the time you wish to delete it, right?

Reply:
@AleksandrAksenov Never mind, I saw you changed it back.

Reply:
You are right - I accidentally set the wrong order and later changed it to the correct one.